app/views/orders/index.html.erb

```diff
@@ -0,0 +1,111 @@
+<h1>Order Fulfillment</h1>
+
+
+<h2>All Orders</h2>
+<br>
+
+
+<%= link_to(user_orders_path(sort:'all'), :class => "btn btn-default", :id => 'all-button') do %>
+ All Orders
+<% end %>
+<%= link_to(user_orders_path(sort:'pending'), :class => "btn btn-default", :id => 'pending-button') do %>
+  Pending
```

Comment:
I'm guessing you're using the space at the beginning of this line to offset the word "Pending"? That falls into presentation, and would be better handled by CSS.